process/smartContract/hooks/blockChainHook.go

```diff
@@ -127,6 +133,8 @@ func (bh *BlockChainHookImpl) GetUserAccount(address []byte) (vmcommon.UserAccou
 
 // GetStorageData returns the storage value of a variable held in account's data trie
 func (bh *BlockChainHookImpl) GetStorageData(accountAddress []byte, index []byte) ([]byte, error) {
+	defer stopMeasure(startMeasure("GetStorageData"))
```

Comment:
i do not see calling of function startMeasure

Reply:
I see right now what it does :)

Reply:
Was also confused by this. You first see `stopMeasure` and then you also have to know that arguments in a deferred function are evaluated at the defer evaluation. Maybe makes sense for a 2 liner here?

Reply:
I intended to make it a one-liner especially to be commonly used (as a pattern). Can we keep it, perhaps adopt it even throughout the project?